src/plugins/janus_audiobridge.c

```diff
@@ -8738,7 +8768,7 @@ static void *janus_audiobridge_participant_thread(void *data) {
 					locked = TRUE;
 					/* Do not let queue-in grow too much */
 					guint count = g_list_length(participant->inbuf);
-					if(count > QUEUE_IN_MAX_PACKETS) {
+					if((int) count > (QUEUE_IN_MAX_PACKETS + lost_packets)) {
```

Comment:
Once you increase QUEUE_IN_MAX_PACKETS, then there's no reason to make it even less strict like you're doing now IMHO. It will make the internal mixer queue for the participant grow beyond control and cause huge and uncoverable latencies.

Reply:
I tried to set it to default 4 and got pops and cracks. On screen is 20-30% of drops set on output channel.
![image](https://github.com/meetecho/janus-gateway/assets/116317/fd0284af-ed40-4b86-88dd-067a47b4b611)

With 20 QUEUE_IN_MAX_PACKETS I don't have such pops/cracks and I don't get over the limit.